--- base.rb	2009-02-07 02:26:43.000000000 +0100
+++ /var/lib/gems/1.8/gems/twitter-0.4.1/lib/twitter/base.rb	2009-01-30 19:19:37.000000000 +0100
@@ -162,6 +162,7 @@ module Twitter
       form_data = {'status' => status}
       form_data.merge!({'source' => options[:source]}) if options[:source]
       form_data.merge!({'in_reply_to_status_id' => options[:in_reply_to_status_id]}) if options[:in_reply_to_status_id]
+      form_data.merge!({'in_reply_to_user_id' => options[:in_reply_to_user_id]}) if options[:in_reply_to_user_id]
       Status.new_from_xml(request('statuses/update.xml', :auth => true, :method => :post, :form_data => form_data))
     end
     alias :update :post
