--- lib/twitter/base.rb~	2009-08-10 14:10:17.000000000 +0200
+++ lib/twitter/base.rb	2009-08-10 14:10:17.000000000 +0200
@@ -20,6 +20,11 @@
       perform_get('/statuses/user_timeline.json', :query => query)
     end
     
+    # added by @dagobart @ 20090810
+    def public_timeline(query={})
+      perform_get('/statuses/public_timeline.json', :query => query)
+    end
+          
     def status(id)
       perform_get("/statuses/show/#{id}.json")
     end
